ideascube/serveradmin/tests/test_views.py

```diff
@@ -642,3 +642,29 @@ def test_staff_forgets_non_existing_connection(mocker, staffapp):
     assert u'No known Wi-Fi networks' not in res.unicode_body
     assert u'my home network' in res.unicode_body
     assert u'random open network' in res.unicode_body
+
+
+@pytest.mark.usefixtures('installed_packages_catalog')
+def test_staff_should_see_packages_with_cards(staffapp, systemuser):
+    from ideascube.configuration import set_config
+    set_config('ideascube', 'displayed-package-ids', ['test.package1'], systemuser)
+    res = staffapp.get(reverse("server:cards"), status=200)
+    assert 'Test package1' in res.unicode_body
+    assert 'Test package2' in res.unicode_body
+    assert 'Test package3' not in res.unicode_body
+
+    form = res.forms['cards']
+    inputs = form.fields['visible']
+    for _input in inputs:
+        should_be_checked = (_input._value == 'test.package1')
+        assert (should_be_checked == _input.checked)
+
+    # inverse the checked
+    for input_ in form.fields['visible']:
```

Comment:
> input is a builtin. I prefer to avoid name clash.

True, although it is such a rarely used one in the context of a webapp that I don't think it matters one bit in this case.

Leading underscores by convention indicate private attributes, which doesn't make any sense in the local scope, so that seems a bit weird to me. And it might trip over programmers new to python but coming from other languages where `self.attribute` is directly accessible as `attribute`. (which is something I personally detest)

Anyway, it doesn't matter much.